addons/survey/static/src/js/survey_form.js

```diff
@@ -614,26 +614,27 @@ publicWidget.registry.SurveyFormWidget = publicWidget.Widget.extend({
     */
     _initDateTimePicker: function ($dateGroup) {
         var disabledDates = []
+        var questionType = $dateGroup.find('input').data('questionType')
```

Comment:
missing `;`
It seems, your linter is still on holiday

Reply:
If you look at the lines above and below, it is missing too. I though it was voluntary

Reply:
my linter was waiting for yours on a beach, to play volley-ball and sipping cocktails.

Reply:
You can take to opportunity to add them :)